test/e2e/playwright/transfer-orders.spec.ts

```diff
@@ -141,7 +141,8 @@ test.describe('Transfer Orders', () => {
     await page.locator(`#court-selection-${orderId}-expand`).click();
     const court = 'manhattan';
     await page.locator(`#court-selection-${orderId}-combo-box-input`).fill(court);
-    await page.getByTestId(`combo-box-option-Southern District of New York (Manhattan)`).click();
+    // await page.getByTestId(`combo-box-option-Southern District of New York (Manhattan)`).click();
+    await page.locator(`[data-value='081']`).click();
```

Comment:
I don't think this is specific enough.  It needs an extra qualifier such as a class or id of a parent in order to namespace it.

Reply:
Now that I have corrected the ComboBox data-testid and made them unique I should be able to target that here instead

Reply:
We should probably remove that commented out line just above too.